api/python/provisioner/commands/set_swupgrade_repo.py

```diff
@@ -15,15 +15,15 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 import logging
-import os
 from pathlib import Path
 
+from provisioner.pillar import PillarKey, PillarResolver
+
+from ..salt import copy_to_file_roots, local_minion_id
 from .set_swupdate_repo import SetSWUpdateRepo
 from .. import inputs, values
-from ..config import (REPO_CANDIDATE_NAME, SW_UPGRADE_REPOS, YUM_REPO_TYPE,
-                      PRVSNR_USER_FILES_SWUPGRADE_REPOS_DIR, RELEASE_INFO_FILE,
-                      ReleaseInfo
-                      )
+from ..config import (REPO_CANDIDATE_NAME, IS_REPO_KEY, RELEASE_INFO_FILE,
```

Comment:
please wrap that into multiple lines so it can be simpler to understand changes in future

Reply:
In my opinion, that approach wastes more space. But I fixed that.